constraints.txt

```diff
@@ -12,13 +12,13 @@ colorama==0.3.9
 cytoolz==0.9.0.1
 decorator==4.3.0
 Deprecated==1.2.5
-eth-abi==1.3.0
+eth-abi==1.2.2
```

Comment:
Why are we downgrading packages here?

Reply:
No, I didn't have to downgrade.  I was mistakenly following `requirements-dev.txt` in `raiden-contracts`.